JAVA/Criminal Management Application/README.md

```diff
@@ -0,0 +1,151 @@
+along with photograph. To identify any Criminal we need some identification
+regarding person. We are developing this software to make this process easier
+than the traditional storage systems. In the modern era when everything is
+computerized, a Criminal Management system is very much in need so as to store
+data of dangerous criminals. 
+</p>
+
+## Introduction
+### Purpose of this project
+<p>
+  The main purpose of our project is automating the traditional way of storing criminal records.
+Here, we could add, update and delete criminal records. We could also add, update and delete
+FIR details which are being filed down manually traditionally. We store all the data into a
+database from where we could use it further. Thus this project provides a very user friendly
+interface for the officials to store and manage the criminal records. 
+</p>
+
+### Project Objective
+<p>
+  This project intends to provide an effective way for storing, retrieving and managing criminal
+records. It also aims at providing a user friendly interface for effortless interaction with the
+user. 
+</p>
+
+### Project Scope
+<p>
+  This project is confined to storing, retrieving and managing criminal records. In further
+developments of the project, we could integrate it with AI and make features such as face
+detections.
+</p>
+
+## UML Diagram
+For UML Digram Click -> <a href="https://github.com/Kumar-laxmi/Criminal-Management-System/blob/main/UML%20Diagram/UML.pdf">UML Diagram</a>
+
+## ER Diagram
+For ER Diagram Click -> <a href="https://github.com/Kumar-laxmi/Criminal-Management-System/blob/main/ER%20Diagram/ER%20Diagram%20-%20Criminal%20Management%20System.png">ER Diagram</a>
+
+## Technology Used
+### IDE:
+- ![Eclipse](https://img.shields.io/badge/Eclipse-2C2255?style=for-the-badge&logo=eclipse&logoColor=white)
+- ![VS Code](https://img.shields.io/badge/Visual_Studio_Code-0078D4?style=for-the-badge&logo=visual%20studio%20code&logoColor=white)
+- ![NetBeans IDE](https://img.shields.io/badge/NetBeansIDE-1B6AC6.svg?style=for-the-badge&logo=apache-netbeans-ide&logoColor=white)
+
+### Language:
+- ![Java](https://img.shields.io/badge/Java-ED8B00?style=for-the-badge&logo=java&logoColor=white)
+
+### Frame-Work:
+- ![Gradle](https://img.shields.io/badge/gradle-02303A?style=for-the-badge&logo=gradle&logoColor=white)
+
+
+### Database:
+- ![Database](https://img.shields.io/badge/MySQL-00000F?style=for-the-badge&logo=mysql&logoColor=white)
+
+## Run Locally
+
+Clone the project
+
+```bash
+  git clone https://github.com/Kumar-laxmi/Criminal-Management-System
+```
+
+Go to the project directory
+
+```bash
+  cd Criminal-Management-System/src/CriminalManagementSystem
+```
+
+Run the application
+
+```bash
+  javac LogIn.java
+```
+
+## Demo Video
+For Demo Video click the link below
+<a href="https://amritacampuschennai-my.sharepoint.com/:v:/g/personal/ch_en_u4cse20039_ch_students_amrita_edu/EZ4vFeSb3z9GgJun-3uPMO4Bj3CWXp8YJhfYP-EjYOE1eA">Click Here</a>
```

Comment:
Plagiarised.
Aim of the project is to motivate people to build projects on their own.

Reply:
And that is not plagarism

Reply:
@Kumar-laxmi Is this your project exhibition code or something ?

Reply:
The thing is recently I have learned JAVA SWINGS so I was trying this for a long time, it is just for practice

Reply:
And to clarify this is not plagarism, it is purely my hardwork

Reply:
Okay but 3lakh+ lines of code in 19hours seems a bit off.

Reply:
Bro I did this using NETBEANS IDE, and using NETBEANS obvious you get so many lines, that is normal

Reply:
@abhijeet007rocks8 And also you may see MYSQL Connector so that also is taking too many lines

Reply:
@abhijeet007rocks8 In bin folder you can see that many ```.class``` files are generated due to multiple testings

Reply:
What ever I wrote is in src folder rest are mostly auto-generated